internal/handlers/pg/msg_explain.go

```diff
@@ -79,6 +79,9 @@ func (h *Handler) MsgExplain(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
+	// if filter was set, it means that pushdown was done
+	pushdown := queryPlanner.HasByPath(types.NewPath("Plan", "Filter"))
```

Comment:
I'm not sure if it's enough. Having a filter doesn't guarantee that pushdown was set. 
Pushdown was set if `pgdb.prepareWhereClause` returned a non-empty string. Otherwise, it wasn't.

Reply:
At the beginning I've started with this approach, but the fact that we need to return another value just for pushdowns (while it's unused in many places) bothered me a bit. I realised, that the `Plan.Filter` from PostrgreSQL EXPLAIN command shows exactly what filter (everything after WHERE clause) what passed, and most importantly - it is already provided to the msg_explain handler.

I don't understand in what cases we can't guarantee that the pushdown is not detected correctly with this approach

Reply:
Hmmm... Ok, got it!

Looks like it works! Then I like the simplicity of your choice! ;)


Will leave an example for more context. 

With `where`:
```
EXPLAIN SELECT * FROM test."ArrayStrings_7a18b54c" WHERE _jsonb='{}';
[
  {
    "QUERY PLAN" : "Seq Scan on \"ArrayStrings_7a18b54c\"  (cost=0.00..27.00 rows=7 width=32)"
  },
  {
    "QUERY PLAN" : "  Filter: (_jsonb = '{}'::jsonb)"
  }
]
```

Without `where`:

```
EXPLAIN SELECT * FROM test."ArrayStrings_7a18b54c";

Seq Scan on "ArrayStrings_7a18b54c"  (cost=0.00..23.60 rows=1360 width=32)
```